packages/docker-compose/build.sh

```diff
@@ -0,0 +1,22 @@
+COMPOSE_VERSION="2.2.2"
+TERMUX_PKG_HOMEPAGE=https://github.com/docker/compose
+TERMUX_PKG_DESCRIPTION="Compose is a tool for defining and running multi-container Docker applications."
+LIBNETWORK_COMMIT=e2f33af831ae9edcfa8efd53dfe24a36337a8250
+TERMUX_PKG_SHA256=001cf72f6bc8a8c43d100389e0bbd3d4d5f5c523f4e3f7ddd53f6a4cd2d6cb18
+TERMUX_PKG_VERSION="2.2"
```

Comment:
Please use TERMUX_PKG_VERSION=2.2.2, and remove COMPOSE_VERSION, there doesn't seem to be any reason for keeping version at 2.2